src/gamecontroller.cs

```diff
@@ -405,6 +514,9 @@ interface GCMicroGamepadSnapshot {
 		[Export ("initWithController:snapshotData:")]
 		IntPtr Constructor (GCController controller, NSData data);
 
+		[Deprecated (PlatformName.MacOSX, 10, 15, message: "Use 'GCController.ControllerWithMicroGamepad()' instead.")]
+		[Deprecated (PlatformName.iOS, 13, 0, message: "Use 'GCController.ControllerWithMicroGamepad()' instead.")]
+		[Deprecated (PlatformName.TvOS, 13, 0, message: "Use 'GCControler.ControllerWithMicroGamepad()' instead.")]
```

Comment:
That's not the managed name you have given (missing `Get` prefix)
Please review all attributes (since it was copy/pasted in many places)